modules/applications-create-using-cli-template.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * applications/creating-new-applications.adoc
+
+[id="applications-create-using-cli-template-{context}"]
+= Creating an application from a template
+
+You can create an application from a previously stored template or from a
+template file, by specifying the name of the template as an argument. For
+example, you can store a sample application template and use it to create an
+application.
+
+.Procedure
+
+* Create an application from a stored template:
++
+----
+$ oc create -f examples/sample-app/application-template-stibuild.json
+$ oc new-app ruby-helloworld-sample
+----
+
+* To directly use a template in your local file system, without first storing it
+in {product-title}, use the `-f|--file` argument:
++
+----
+$ oc new-app -f examples/sample-app/application-template-stibuild.json
+----
+
+*Template Parameters*
+
+* When creating an application based on a template, use the
+`-p|--param` argument to set parameter values defined by the template:
```

Comment:
s/parameter values defined/the parameter values that are defined

I think that there are some hidden assumptions here.